### PR TITLE
Enhance discovery setup

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: roles/plugins/foreman_discovery/tasks/main.yml [hunk 1]
```diff
@@ -4,4 +4,16 @@
...
     plugin_installer_options: >
       --enable-foreman-plugin-discovery
+      --enable-foreman-proxy-plugin-discovery
       {{ foreman_discovery_installer_options }}
+
+# TODO until http://projects.theforeman.org/issues/22479 is fixed
+- name: 'check if FDI tar file exists'
+  stat: path=/var/lib/tftpboot/boot/fdi-image-latest.tar
+  register: fdi_image_tar_file
+- name: 'Install latest FDI image'
```
Comment (ehelms) [inline roles/plugins/foreman_discovery/tasks/main.yml]: Mind an empty line above this to bring it in line with other task files?

Comment (ares) [reply roles/plugins/foreman_discovery/tasks/main.yml]: I wanted to keep the together as the comment applies to both tasks, should I copy the comment then?

Comment (ehelms) [reply roles/plugins/foreman_discovery/tasks/main.yml]: Maybe move to a separate file then and include it here? `hacks.yaml` or `workarounds.yaml` ? Will make it clearer to me and allow keeping conventions.

Comment (ekohl) [reply roles/plugins/foreman_discovery/tasks/main.yml]: I'd prefer to get the bug fixed instead.

Comment (ares) [reply roles/plugins/foreman_discovery/tasks/main.yml]: I see you already sent a PR, if that gets merged, I'm happy to remove this workaround, though I think it shouldn't block this PR if it's not merge shortly

File: roles/plugins/foreman_discovery/tasks/main.yml [hunk 2]
```diff
@@ -4,4 +4,9 @@
   vars:
     plugin_installer_options: >
       --enable-foreman-plugin-discovery
+      --enable-foreman-proxy-plugin-discovery
       {{ foreman_discovery_installer_options }}
+
+# TODO until http://projects.theforeman.org/issues/22479 is fixed
```
Comment (ekohl) [inline roles/plugins/foreman_discovery/tasks/main.yml]: This is now fixed in nightlies

**Comment by ares** [comment]
@ekohl removed the workaround, should be ready for merge

**Comment by ekohl** [comment]
Isn't there an option to download images (that defaults to false)?

**Comment by ares** [comment]
yeah sorry, I only deleted the workaround without re-adding the parameter, should be good now

**Comment by ares** [comment]
it's a bit frustrating to see something is approved but not merged :-)

**Comment by ekohl** [comment]
If there are more reviewers on a PR I tend to give them a chance to have another look. @ehelms?

**Comment by ares** [comment]
Sure, though their comments were addressed already :-) Also we have 3 acks at https://github.com/theforeman/forklift/pull/614 and 4th reviewer said LGTM **https://github.com/theforeman/forklift/pull/614#issuecomment-367993619

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**